backend/compact-connect/lambdas/license-data/handlers/providers.py

```diff
@@ -0,0 +1,97 @@
+# pylint: disable=unused-argument,unexpected-keyword-arg,missing-kwoa
+# Pylint really butchers these function signatures because they are modified via decorator
+# to cut down on noise level, we're disabling those rules for the whole module
+import json
+
+from aws_lambda_powertools.utilities.typing import LambdaContext
+
+from exceptions import CCInvalidRequestException
+from handlers.utils import api_handler
+from config import config, logger
+
+
+@api_handler
+def query_providers(event: dict, context: LambdaContext):  # pylint: disable=unused-argument
+    """
+    Query providers data
```

Comment:
I'm trying to think of the best way to handle this. The `event` object is pretty big but is a standard payload provided to a lambda by API Gateway. An example event is included in the test resources (`tests/resources/api-event.json`). For this specific endpoint, the actual POST body schema is already documented for the API spec and validation over [here](https://github.com/InspiringApps/JCC-licensure-compact-system/blob/26240cc736469ced2f7e3b8b76c862594d44690c/backend/compact-connect/stacks/api_stack/query_providers.py#L89). I could add some comments or doc-string details to this method that spell that all out, but I'm hesitant to do that because that's exactly the kind of comment that is very easy to forget to update as the code evolves, leaving misleading information hidden in the code-base. What's your preference, here?

Reply:
Makes complete sense. What about saying two things:

* that `event` is a standard payload provided to a lambda by API gateway
* that the shape of `query` is validated in `api_stack/query_providers.py` (Or, if you'd like to avoid referring to a specific file, perhaps "in the Query Providers AWS Stack"

Does this feel reasonable to you? (Totally OK if not, just brainstorming.)

Reply:
Yeah, that seems reasonable. 👍